package.json

```diff
@@ -13,12 +13,13 @@
     "url": "https://github.com/xpring-eng/Xpring-JS/issues"
   },
   "scripts": {
-    "build": "tsc -d",
-    "clean": "rm -rf ./build",
+    "build": "./scripts/regenerate_protos.sh && tsc -d",
```

Comment:
this seems fine, but do you think it's worth having an explicit build script? i feel like if we add one more step it would be nice to encapsulate it

Reply:
I think this will shake out when this library converges with Xpring-web.